Source/WebCore/page/LoginStatus.h

```diff
@@ -37,25 +37,29 @@ namespace WebCore {
 class LoginStatus {
     WTF_MAKE_ISO_ALLOCATED_EXPORT(LoginStatus, WEBCORE_EXPORT);
 public:
+
     static constexpr uint32_t UsernameMaxLength = 64;
+    static constexpr Seconds TimeToLiveAuthentication { 30_s };
     static constexpr Seconds TimeToLiveShort { 24_h * 7 };
     static constexpr Seconds TimeToLiveLong { 24_h * 90 };
 
     enum class CredentialTokenType : bool { LegacyCookie, HTTPStateToken };
     enum class AuthenticationType : uint8_t { WebAuthn, PasswordManager, Unmanaged };
 
-    WEBCORE_EXPORT static ExceptionOr<UniqueRef<LoginStatus>> create(const RegistrableDomain&, const String& username, CredentialTokenType, AuthenticationType);
-    WEBCORE_EXPORT static ExceptionOr<UniqueRef<LoginStatus>> create(const RegistrableDomain&, const String& username, CredentialTokenType, AuthenticationType, Seconds timeToLive);
+    WEBCORE_EXPORT static ExceptionOr<UniqueRef<LoginStatus>> create(const RegistrableDomain&, const String&, CredentialTokenType, AuthenticationType);
```

Comment:
We can keep the name for username, since the name adds more information than just the type.